src/RelayConnection.ts

```diff
@@ -71,7 +71,7 @@ export default class RelayConnection<T extends Entity> {
       if (this.args.last < 0) {
         throw new LimitError("last");
       }
-      return slice.slice(slice.length - this.args.last);
```

Comment:
So if `slice.length` was 8 and `this.args.last` was 10, this would previously do a `slice(-2)`. Got it.

Would you add a test case for this? Thanks for tracking it down!